kong/timing/init.lua

```diff
@@ -62,11 +70,13 @@ function _M.auth()
     return
   end
 
-  assert(ngx.ctx.req_trace_id == nil)
+  local ngx_ctx = ngx.ctx
+  
+  assert(ngx_ctx.req_trace_id == nil)
 
-  local http_x_kong_request_debug = ngx_var.http_x_kong_request_debug
-  local http_x_kong_request_debug_token = ngx_var.http_x_kong_request_debug_token
-  local http_x_kong_request_debug_log = ngx_var.http_x_kong_request_debug_log
+  local http_x_kong_request_debug = get_header("X-Kong-Request-Debug", ngx_ctx)
```

Comment:
lower and replace dashes the header's name (`X-Kong-Request-Debug`) will be a little faster, see the implementation of `get_header`

Reply:
I lowered all of them and converted dashes to underscores.